php-checks/src/main/java/org/sonar/php/checks/CheckList.java

```diff
@@ -221,7 +221,8 @@ private CheckList() {
       VariableVariablesCheck.class,
       VarKeywordUsageCheck.class,
       WeakSSLProtocolCheck.class,
-      WrongAssignmentOperatorCheck.class);
+      WrongAssignmentOperatorCheck.class,
+      UseOfOctalValueCheck.class);
```

Comment:
It would be better to add this line at the right position to keep alphabetical order. That makes it unlikely to have a conflict when several PRs add a line at the same position.